tests/manager/openAccount/managerCanOpenAccount.spec.js

```
@@ -28,4 +49,24 @@ test('Assert manager can add new customer', async ({ page }) => {
   1. Do not rely on the customer row id for the step 13. 
     Use the ".last()" locator to get the last row.
   */
+openAccountPage = new OpenAccountPage(page);
+await addCustomerPage.clickOpenAccountsButton();
+await openAccountPage.waitForLoading();
+const customerName = `${customerFirstName} ${customerLastName}`;
+await openAccountPage.selectCustomerName(customerName);
+
+const currency = faker.helpers.arrayElement(['Dollar', 'Pound', 'Rupee']);
+await openAccountPage.selectCurrency(currency);
+
+//await openAccountPage.selectCurrency('Dollar');
+await openAccountPage.clickProcessButton();
+await page.reload();
```

Same as earlier: avoid `await page.reload();` after processing. Handle the Process dialog explicitly and then navigate to Customers via the UI, or wait for an element that indicates the app is ready. Reload introduces flakiness and may mask failures.